Logger/LogFactory.cs

```diff
@@ -1,11 +1,28 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public class LogFactory
     {
+        private string filePath;
```

Comment:
Fields should be underscore PascalCase
```suggestion
        private string _FilePath;
```